src/parser/paladin/holy/CombatLogParser.js

```diff
@@ -82,8 +86,8 @@ class CombatLogParser extends CoreCombatLogParser {
     hammerOfWrath: HammerOfWrath,
 
     // Generic healer things
-    manaLevelChart: ManaLevelChart,
-    manaUsageChart: ManaUsageChart,
+    // manaLevelChart: ManaLevelChart,
+    // manaUsageChart: ManaUsageChart,
 
     // Features
```

Comment:
delete

Reply:
then i just have to re-add it when blizzard finally learns about mana and paladin

Reply:
Correct